src/enrichment/babamul.rs

```diff
@@ -186,6 +168,12 @@ impl Babamul {
                 .push(alert);
         }
 
+        // If there is nothing to send, return early
+        if alerts_by_topic.is_empty() {
+            info!("No LSST alerts to send to Babamul");
```

Comment:
once we are happy with the code, we could move that info to trace or debug, so they don't pollute the logs

Reply:
True, but it also might be nice to see some confirmation that they were at least processed, even though none will make it to Kafka.